src/global-messages/types.ts

```diff
@@ -11,6 +11,7 @@ export enum GlobalMessageContextEnum {
   appTicketing = 'app-ticketing',
   appPurchaseOverview = 'app-purchase-overview',
   appPurchaseConfirmation = 'app-purchase-confirmation',
+  appPurchaseInformation = 'app-purchase-information',
```

Comment:
Hard to intuitively understand where 'app-purchase-information' is located in the app. Maybe something like 'app-purchase-confirmation-bottom' would be more descriptive?

Reply:
Updated